README.md

```diff
@@ -317,6 +317,48 @@ Stop the running container:
 
     docker stop ipfs_host
 
+### Building Docker images
+
+For other platforms such as ARM, you may need to build a docker image to run.
+
+The main Dockerfile takes the following args:
+- `base`: The base go image for the target platform
+- `busybox_base`: The busybox base image for the target platform
+- `tini_executable`: The tini executable path for the target platform
+- `glibc_shared_lib_arch`: The target platform shared lib for glibc supplementing busybox
```

Comment:
Is there really no better way to do this (i.e., with one variable). From what I can tell, there simply isn't but I'm not a Docker expert so I'd like to double check.

Reply:
Same here. This was seemingly the only way.